web/src/resize.ts

```diff
@@ -89,10 +92,10 @@ export function reset_compose_message_max_height(bottom_whitespace_height) {
         bottom_whitespace_height = get_bottom_whitespace_height();
     }
 
-    const compose_height = $("#compose").get(0).getBoundingClientRect().height;
+    const compose_height = $("#compose").get(0)?.getBoundingClientRect().height ?? 0;
     const compose_textarea_height = Math.max(
-        $("textarea#compose-textarea").get(0).getBoundingClientRect().height,
-        $("#preview_message_area").get(0).getBoundingClientRect().height,
+        $("textarea#compose-textarea").get(0)?.getBoundingClientRect().height ?? 0,
+        $("#preview_message_area").get(0)?.getBoundingClientRect().height ?? 0,
```

Comment:
These should probably be `!`s or asserts, since we strongly expect these elements to be in the DOM? I think we might prefer a crash over returning `0`.

Reply:
Yeah that makes sense. Updated!